source/exe/process_wide.cc

```diff
@@ -13,8 +13,14 @@
 #endif
 
 namespace Envoy {
+namespace {
+// Static variable to sanity check init state.
+bool process_wide_iniitalized;
+} // namespace
 
 ProcessWide::ProcessWide() {
+  ASSERT(!process_wide_iniitalized);
```

Comment:
isn't process_wide_iniitalized uninitialized at this point?

Reply:
It's a static variable, so gets zero init. It's now a `uint32_t` as well, due to how `main_common_test` works.